infinitered/ReactNativeEssentials
# Chapter 9: Internationalize All The Things

### Summary

Chapter 9: Internationalize All The Things

This PR implements i18n.

### How to verify

1. `yarn`
2. `yarn start`
3. `i` & `a`
4. change language to Arabic and confirm it works

**Expected results**: app is available in `en` and `ar`

### Screenshots

| AR                                             | EN                                             |
| ----------------------------------------------- | --------------------------------------------------- |
| <img width="300" alt="before screenshot" src="https://github.com/user-attachments/assets/4e996e53-65a2-4a22-9699-c2d3b16d104d" /> | <img width="300" alt="after screenshot" src="https://github.com/user-attachments/assets/6bf5c7be-5a57-4430-aae2-888f914606a4" /> |
